### PR TITLE
fix(v2): fix build caused when one tab item

### PR DESCRIPTION
**Pull request by lex111**
## Motivation

Hotfix: after the .32 release, if there is one code tab, then the build command will fail.

Actually, my fault, this error didn’t exist when the devserver was working, but something did not work when compiling the build :(
Probably we need tests that the build command checks.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

See #1934 and apply build.

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: packages/docusaurus-theme-classic/src/theme/Tabs/index.js [hunk 1]
```diff
@@ -31,7 +31,11 @@ function Tabs(props) {
         ))}
       </ul>
       <div className="margin-vert--md">
-        {[...children].filter(child => child.props.value === selectedValue)[0]}
+        {
```
Comment (yangshun) [inline packages/docusaurus-theme-classic/src/theme/Tabs/index.js]: How about we use `React.Children` instead? Better to stick to what React provides

Comment (lex111) [reply packages/docusaurus-theme-classic/src/theme/Tabs/index.js]: oh yeah, fixed (although `Array.from` is a bulletproof solution), probably it was necessary to use it initially, but I prevented it :(

**Comment by lex111** [comment]
Will there be 2.0.0-alpha.33 for this?

**Comment by yangshun** [comment]
Personally, I don't think it's very urgent because only users having one item will run into this and they can choose not to upgrade to 2.0.0-alpha.22 yet. @endiliey is our release guy and he could release one if it isn't too much work.

**Pull request merged**